src/controllers/AnnotationModeController.js

```diff
@@ -590,12 +603,7 @@ class AnnotationModeController extends EventEmitter {
         const pageEl = this.isMobile
             ? this.container
             : this.annotatedElement.querySelector(`[data-page-number="${pageNum}"]`);
-        let popoverLayer = pageEl.querySelector('.ba-dialog-layer');
-        if (!popoverLayer) {
-            popoverLayer = document.createElement('span');
-            popoverLayer.classList.add('ba-dialog-layer');
-            pageEl.appendChild(popoverLayer);
-        }
+        getPopoverLayer(pageEl);
```

Comment:
Looks like `getPopoverLayer` implicitly creates the layer if it doesn't currently exist. Could we refactor out the create aspect to it's own util function and call that here to be more semantically correct?